tui/common.go

```diff
@@ -327,6 +327,19 @@ func (page *BasePage) setup(tui *Tui, id int, btns int, returnID int) {
 		"Use [Tab] or the arrow keys [up and down] to navigate", Fixed)
 
 	page.window.SetVisible(false)
+
+	// Escape-key cancel's this screen and returns
+	// same as the default 'Cancel' button
+	page.window.OnKeyDown(func(ev clui.Event, data interface{}) bool {
```

Comment:
looks good and works fine, I have only one question, what would be the expected behavior if you're typing in a text field and press escape?

Reply:
The same as pressing the Cancel button.
If there are text fields where we think this is a concern/risk, we can trap and remove the Escape from the input stream.
But we will no be using vi/emacs like editing in our text fields. :)

Reply:
I would rather make it Ctrl+Escape though